src/main/java/digital/vix/vertxdemo/utils/ApplicationUtils.java

```diff
@@ -0,0 +1,19 @@
+package digital.vix.vertxdemo.utils;
+
+public class ApplicationUtils {
+
```

Comment:
Looks ok to me. ApplicationUtils is very wide term. It could be more specific like ArrayUtils.

Reply:
Good idea. Will Change